spec/tapioca/dsl/compilers/sidekiq_worker_spec.rb

```diff
@@ -8,9 +8,25 @@ module Dsl
     module Compilers
       class SidekiqWorkerSpec < ::DslSpec
         describe "Tapioca::Dsl::Compilers::SidekiqWorker" do
+          sig { void }
+          def backup_activesupport
+            Object.const_set(:ActiveSupportBackup, Object.send(:remove_const, :ActiveSupport)) # rubocop:disable RSpec/RemoveConst
+          end
```

Comment:
I don't have a strong opinion, if `after_teardown` doesn't run somehow we probably have more important issues than not resetting `ActiveSupport`. I'll leave it to @amomchilov

Reply:
I also couldn't figure out how to run a single or a subset of tests, that would be quite useful because right now I can only run the whole test suite which takes quite some time 😞

Reply:
OK it wasn't super straightforward because `active_support` is actually required in `bin/test` (apparently for Rails 6 support?) and so it's already part of the object space at the time of forking. That means I still need to undefine and unload it, but at least I don't need to put it back after the tests have run 👍 

I grabbed some code from `pipeline_spec.rb` to do that and tests are passing locally 🙌